src/components/utils/src/threads/posix_thread.cc

```diff
@@ -151,10 +148,10 @@ Thread::Thread(const char* name, ThreadDelegate* delegate)
     , delegate_(delegate)
     , handle_(0)
     , thread_options_()
-    , isThreadRunning_(false)
     , stopped_(false)
     , finalized_(false)
-    , thread_created_(false) {}
+    , thread_created_(false)
```

Comment:
@anosach-luxoft , thread_created_ is need in case some call start() couple times.